Dockerfile

```diff
@@ -43,9 +51,11 @@ RUN apt-get install -y \
       mesa-utils \
       netpbm \
       pkg-config \
+      rsync \
       tcsh \
       unzip \
       vim \
+      wget \
```

Comment:
Seems unnecessary to install wget, if curl is already available.

Reply:
Resolved in b5b1714